dwl.c

```diff
@@ -1846,6 +1865,10 @@ main(int argc, char *argv[])
 		exit(EXIT_FAILURE);
 	}
 
+	if (!drop_permissions()) {
```

Comment:
I would guess from the error posted over at #7 that root is needed at least until `wlr_backend_autocreate` returns.